### ISSUE TITLE
Home Connect API error: client has limited user list - user not assigned to client [invalid_client]

### ISSUE THREAD

**Issue by S-Mario987** [thoukydides/homebridge-homeconnect#115]
Hi,

i get the following Error Message:

Home Connect API error: client has limited user list - user not assigned to client [invalid_client]

what should I need to do? 😊

thanks for your help!

**Comment by thoukydides** [comment]
Errors with the "Home Connect API" prefix are generally returned by the Home Connect servers.

Documentation for the Home Connect API errors can be found on in the Home Connect API Docs at https://api-docs.home-connect.com/general?#api-errors and https://api-docs.home-connect.com/authorization?#authorization-errors.

In this case the documentation says:

| Error Cause | Code | Error | Error Description |
| -- | -- | -- | -- |
| Invalid HC account - please check configured test account | 302 | unauthorized_client | client has limited user list - user not assigned to client |

So, basically, you have incorrectly configured the Home Connect application on https://developer.home-connect.com/applications.

Most likely you created the Home Connect developer account using a different email address than the one for your Home Connect appliances. Alternatively, you have put an incorrect email address in either the "Default Home Connect User Account for Testing" field of your profile or the "Home Connect User Account for Testing" field of the application.

**Comment by S-Mario987** [comment]
Hi,

thank you very much for your message 😊


I have looked again and it is exactly the same.

Everything worked fine until now. Only since a few days I have the problem

**Comment by thoukydides** [comment]
In that case it might be a problem with the Home Connect servers. I suggest you use ask at https://developer.home-connect.com/support/contact

Have you recently converted your Home Connect account to a SingleKey ID by any chance?

I have not seen that particular error with any of my clients, so it does not appear to be a general problem.

**Comment by S-Mario987** [comment]
Hi,

I have switched to Single Key ID. 

I have changed the development more often now, even created a new developer account with a different mail address.

When I create a new development, then it goes about a day and then comes the same error message. 

I have reinstalled the plugin several times. I really do not know what to do 😥

**Comment by thoukydides** [comment]
Have you tried contacting the Home Connect developer support at https://developer.home-connect.com/support/contact as I suggested?
